compact/hcal/hcal_backward.xml

```diff
@@ -54,8 +54,15 @@
   <!--  Definition of the readout segmentation/definition  -->
   <readouts>
     <readout name="HcalEndcapNHits">
-      <segmentation type="CartesianGridXY" grid_size_x="100 * mm" grid_size_y ="100 * mm"/>
-      <id>system:8,barrel:3,module:4,layer:8,slice:5,x:32:-16,y:-16</id>
+      <!--<segmentation type="PolarGridRPhi2"
+      grid_r_values="HcalEndcapN_rmin 90.9878*cm 100.89*cm 111.395*cm 122.516*cm 134.229*cm 146.58*cm 159.546*cm 173.155*cm 187.424*cm 202.377*cm 218.019*cm 234.353*cm 251.444*cm HcalEndcapN_rmax"
+      grid_phi_values="6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg"
+      offset_phi="0.0*deg"/>-->
```

Comment:
Is there a reason to keep this around?
```suggestion
```

Reply:
I updated the formatting and fixed the number of phi bins from 36x6*cm to 60x6*cm